roles/executor/templates/env

```diff
@@ -0,0 +1,2 @@
+APIMON_IMAGE={{ executor_image }}
+APIMON_SCHEDULER_IMAGE={{ scheduler_image }}
```

Comment:
not needed

Reply:
ok, separated the env file per role, this way each role has its own env file and its own image variable